Source/Modules/php.cxx

```diff
@@ -2007,6 +2013,10 @@ class PHP : public Language {
 	  if (value) {
 	    Printf(pragma_phpinfo, "%s\n", value);
 	  }
+	} else if (Strcmp(type, "version") == 0) {
+	  if (value) {
+	    Printf(pragma_version, "%s", value);
```

Comment:
Hey, You suggested initialising pragma_version to NULL by default and then assign the value to it here. Probably did you mean pragma_version pointing to where value is pointing to, since both are string pointers. Instead of allocating new resource of a String to pragma_version pointer. 
` pragma_version = value `